### ISSUE TITLE
Misspellings are listed as compile error during PDF/UA export

### ISSUE THREAD

**Issue by eltos** [typst/webapp-issues#923]
### Description

When exporting to PDF/UA-1 fails, e.g. because of missing alt texts, the complie errors list includes misspellings. This is redundant as the misspellings are already listed in the dedicated misspellings section. It also makes it very difficult to spot the actual error, especialy if there are many spelling errors, which is likely to happen since the list includes misspellings from unrelated/unused files in the project directory.

In the following example, only the two PDF/UA-1 compiler errors should be visible (the ones marked with a blue dot). Notice that the section heading correctly displays "2 compiler errors". However, the list shows 5 "errors" including misspellings.

<img width="852" height="791" alt="Image" src="https://github.com/user-attachments/assets/9ab7569e-da26-4856-86cb-73d2160fd233" />

### Browsers

Firefox

### OS

Windows
